sources/packages/web/src/views/institution/locations/offerings/OfferingEdit.vue

```diff
@@ -62,14 +62,14 @@
     </template>
     <offering-form-submit
       submitLabel="Update offering now"
-      :data="initialData"
+      :offeringId="offeringId"
```

Comment:
Please look at the consideration given below. for OfferingForm We need to optionally allow the data either or call the service method with location and program in it.

OfferingEdit.vue is used by the institutions.

Hence while editing an offering by an institution, the offering details are loaded by the institution API. Now if the loading of data from API happens inside OfferingForm.vue and hence the API with only offeringId will be called for the institutions client and it is going to return a non 200 response which means it will fail.

Reply:
Thanks for the explanation @dheepak-aot . I will fix this back.